### PR TITLE
divergence from reference results text

### PR DESCRIPTION
**Pull request by EricRLucas**
Addresses #31 
@hardingnj @cclarkson Haven't written much in the results, but then I don't think we felt that much needed to be written.

### CODE REVIEW

**Comment by cclarkson** [comment]
Couple of suggestions:

Maybe rephrase "broadly equally similar" to "similarly diverged from the reference genome".

Do we have a reference for the chimeric nature of the PEST reference genome and/or for the divergence between gam/col - i.e. a sentence on why we are not surprised that we are able to use a single reference?

**Comment by EricRLucas** [comment]
The original paper for the first PEST assembly mentions the mixed nature of the reference. I've also added a statement about the possibility that the similarity to coluzzii on the X is due to the fact that the (X-linked) pink-eye mutation that defines the strain came from its coluzzii ancestor, so the fact that the strain was forcibly selected for the link eye phenotype would have removed any gambiae ancestry around it on the X. I don't know whether anyone has made this point before (couldn't find anything about it in the literature) and I haven't been able to find out the position of the pink-eye mutation to check that it is centromeric, so happy to remove that statement if necessary.

**Comment by EricRLucas** [comment]
Thanks Chris. Is the following sentence suitable and accurate?
"""
All analyses of geographical population structure were conducted on euchromatic regions of Chromosome 3, which avoids regions of polymorphic inversions, reduced recombination and unequal differentiation from the reference genome [AG1000G ref].
"""

**Comment by cclarkson** [comment]
sounds good, thanks @EricRLucas

**Comment by EricRLucas** [comment]
as suggested
